addons/binding/org.openhab.binding.roku/src/main/java/org/openhab/binding/roku/internal/RokuState.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.roku.internal;
+
+import java.io.IOException;
+
+import org.eclipse.smarthome.core.library.types.RawType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.openhab.binding.roku.internal.protocol.RokuCommunication;
+
+/**
+ * The {@link RokuState} class defines various pieces of state a Roku device may have that will be translated to
+ * functionality the binding offers.
+ *
+ * @author Jarod Peters - Initial contribution
+ * @auther Shawn Wilsher - Overhaul of channels and properties
+ */
+public class RokuState {
+    public String udn;
+    public String serial_number;
```

Comment:
Can you follow the naming conventions maybe with some annotation to map the fieldname?